format-specs/schema.json

```diff
@@ -11,95 +12,92 @@
+        "required": ["encoding", "geometry_type"],
+        "properties": {
+          "encoding": {
+            "type": "string",
+            "const": "WKB",
+            "description": "Name of the geometry encoding format. Currently only 'WKB' is supported."
+          },
+          "geometry_type": {
+            "oneOf": [
+              {
+                "type": "string",
+                "const": "Unknown"
+              },
+              {
+                "$ref": "#/definitions/geometry_type"
+              },
+              {
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/geometry_type"
                 },
-                {
-                  "type": "number",
-                  "description": "The maximum constant latitude line that bounds the rectangle (ymax)."
-                }
-              ]
+                "uniqueItems": true
+              }
+            ],
+            "description": "The geometry type(s) of all geometries, or 'Unknown' if they are not known."
+          },
+          "crs": {
+            "oneOf": [
+              {
+                "$ref": "https://proj.org/schemas/v0.5/projjson.schema.json"
+              },
+              {
+                "type": "null"
+              }
+            ],
+            "description": "JSON object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude based on the WGS84 datum, and CRS-aware implementations should assume a default value of OGC:CRS84."
+          },
+          "edges": {
+            "type": "string",
+            "enum": ["planar", "spherical"],
+            "description": "Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'."
+          },
+          "orientation": {
+            "type": "string",
+            "const": "counterclockwise",
+            "description": "Winding order of exterior ring of polygons; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order."
+          },
+          "bbox": {
+            "type":"array",
+            "description": "Bounding Box of the geometries in the file, formatted according to RFC 7946, section 5.",
+            "items":{
+              "type":"number"
             },
-            "epoch": {
-              "type": "number",
-              "description": "Coordinate epoch in case of a dynamic CRS, expressed as a decimal year."
-            }
+            "oneOf":[
+              {
+                "description": "2D bbox consisting of (xmin, ymin, xmax, ymax)",
+                "minItems":4,
+                "maxItems":4
+              },
+              {
+                "description": "3D bbox consisting of (xmin, ymin, zmin, xmax, ymax, zmax)",
+                "minItems":6,
+                "maxItems":6
```

Comment:
I wouldn't go this far.  The [GeoJSON spec](https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1) doesn't require (only recommends) 2 or 3 dimensions.

Instead, I would go with this:
```json
"bbox": {
  "type": "array",
  "minItems": 4,
  "items": {
    "type": "number"
  }
}
```

Reply:
Also fine with me, I just reduced what has been discussed in #88...

Reply:
Let's discuss bbox in #88 (maybe leave it as is here so it's not in the diff?)